### PR TITLE
SEO-177700-React-dropped-keywords-issue-fixes

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @gowthamanthavasiyappan ,
Please review the changes.
Thanks.
Regards,
Edith.

### CODE REVIEW

File: ej2-react/heatmap-chart/getting-started.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React HeatMap chart component | Syncfusion
-description:  Checkout and learn about Getting started with React HeatMap chart component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with React HeatMap Chart Component | Syncfusion
+description:  Checkout and learn about getting started with Syncfusion Essential React HeatMap chart component, it's elements and more details.
```
Comment (gowthamanthavasiyappan) [inline ej2-react/heatmap-chart/getting-started.md:4]: Remove unwanted space here

File: ej2-react/multi-select/getting-started.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Multi select component | Syncfusion
-description:  Checkout and learn about Getting started with React Multi select component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with React MultiSelect Component | Syncfusion
+description:  Checkout and learn about getting started with Syncfusion Essential React MultiSelect component, it's elements and more details.
```
Comment (gowthamanthavasiyappan) [inline ej2-react/multi-select/getting-started.md:4]: Remove unwanted space here

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: ej2-react/multi-select/getting-started.md (hunk 2 above)
Comment (Edithabich) [reply ej2-react/multi-select/getting-started.md]: resolved

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Pull request merged**